beagle-recipe/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "beagle-lib" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: git@github.com:beagle-dev/beagle-lib.git
+  git_tag: hmc-clock
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+    - libtool
+    - openjdk
```

Comment:
It always seemed strange to me that the beagle conda required openjdk. Have you tried without?

Reply:
It looks like a JDK is an optional dependency for Beagle so I will remove it